karate-tests/src/test/java/mocks/sandbox.js

```diff
@@ -80,49 +56,46 @@ function invalidValueError(field, value, request) {
     return false
 }
 
-
-/*
-    Response definitions
-*/
-function returnBundle(filename) {
-    response.body = getTimestampedBody(`classpath:mocks/stubs/patientResponses/${filename}`)
-    response.status = 200
-}
-
-
-function returnEmptyBundle() {
-    response.body = getTimestampedBody('classpath:mocks/stubs/patientResponses/empty_bundle.json')
-    response.status = 200
-}
-
-
 function returnMissingValueError(diagnostics) {
-    let body = context.read('classpath:mocks/stubs/patient/errorResponses/missing_value.json')
+    let body = context.read('classpath:mocks/stubs/errorResponses/MISSING_VALUE.json')
     body["issue"][0]["diagnostics"] = diagnostics
     response.body = body
     response.status = 400
+    return false
```

Comment:
Feels like a weird method name for what it returns. Maybe change it to setMissingValueError and use false higher up in the stack? happy to be disagreed with

Reply:
A few similar examples in [supporting-functions.js](https://github.com/NHSDigital/personal-demographics-service-api/pull/995/files#diff-88ffe8f45bf3132e29f5d2c7b197c2385e39a70c4a10e2bcdd4d313512760cbc)

Reply:
Yes I wasn't so comfortable with the function names either. I think setMissingValueError would be preferable.